.travis.yml

```diff
@@ -36,12 +36,14 @@ matrix:
     cache:
       directories:
         - $HOME/.m2/repository
-      before_cache:
-        - rm -rf $HOME/.m2/repository/org/apache/qpid/*
+    before_cache:
+      - rm -rf $HOME/.m2/repository/org/apache/qpid/*
     install: []
     before_script: []
     script:
     - mvn apache-rat:check
+    after_failure:
+    - cat target/rat.txt
```

Comment:
Depending on the failure, the file might not exist, worth guarding?

Reply:
If it does not exist, a nonzero return value from the last after_failure command does not really change anything. To some degree, missing target/rat.txt can be considered an additional failure, because of the expected way the rat CI job is supposed to fail.

I can add the check, it would make the job then future-proof against adding additional commands to after_failure.

Reply:
I wouldn't really see it as an 'additional failure', just a consequence of the actual failure, and so it seemed like any output reported for this failing is just distracting from seeing the actual failure. But I guess its not going to happen often and the output is fairly obvious, and unlike other jobs theres no additional checks here besides rat, so maybe its not as important as when it were part of the overall test run like before.

Reply:
Gonna go with `|| true` there.